test/cmd/basicresources.sh

```diff
@@ -29,10 +29,8 @@ os_git_regex="$( escape_regex "${OS_GIT_VERSION%%-*}" )"
 kube_git_regex="$( escape_regex "${KUBE_GIT_VERSION%%-*}" )"
 etcd_version="$(echo "${ETCD_GIT_VERSION}" | sed -E "s/\-.*//g" | sed -E "s/v//")"
 etcd_git_regex="$( escape_regex "${etcd_version%%-*}" )"
-os::cmd::expect_success_and_text 'oc version' "oc ${os_git_regex}"
-os::cmd::expect_success_and_text 'oc version' "kubernetes ${kube_git_regex}"
-os::cmd::expect_success_and_text 'oc version' "features: Basic-Auth"
-os::cmd::expect_success_and_text 'openshift version' "openshift ${os_git_regex}"
+os::cmd::expect_success_and_text 'oc version' "Client Version: .*GitVersion${os_git_regex}"
```

Comment:
This needs to be:
```
os::cmd::expect_success_and_text 'oc version' "Client Version: .*GitVersion:\"${os_git_regex}"
```
notice the `:\"` between `GitVersion` and `${os_git_regex}`.